HermesProxy/World/Server/WorldSocket.cs

```diff
@@ -687,7 +687,7 @@ void HandleConnectToFailed(ConnectToFailed connectToFailed)
                     break;
                 case ConnectToSerial.WorldAttempt5:
                 {
-                    Log.Print(LogType.Error, "Failed to connect 5 times to world socket, aborting login");
+                    Log.PrintNet(LogType.Error, LogNetDir.S2P, "Failed to connect 5 times to world socket, aborting login");
```

Comment:
I think PrintNet only makes sense to be used for printing opcodes being sent/received, not other errors like that. And S2P which you use here is not even accurate, this error is about client failing to connect to instance socket of proxy.

Reply:
Okay, I updated the commit but left the cases where it is actually network related (like sending or receiving packets).